### PR TITLE
Add a public method to know whether the runtime is configured

### PR DESCRIPTION
**Pull request by stof**
Refs #236

### CODE REVIEW

**Comment by stof** [comment]
Failure in the job `Highest versions of the dependencies` is unrelated to that change. It looks like the hash of the runtime is different for the new version of webpack (they might have altered their runtime a bit).

**Comment by stof** [comment]
Once this is merged and released, we can update the recipe to use it by default.

**Comment by stof** [comment]
and documentation too.

**Comment by Kocal** [comment]
Thanks for the feature, it will really helps with PhpStorm webpack integration. :+1: 

Also I've just seen some linting issues. In the whole file, it use `it(` instead of `it (`, and there is always a blank line between `});` (from `it()`) and `});` (from `describe()`).
I think it would be nice to correct them to keep something consistent. :)

File: test/index.js [hunk 1]
```diff
@@ -414,6 +414,21 @@ describe('Public API', () => {
...
     });
 
+    describe('isRuntimeConfigured', () => {
+
+        it ('should return true if the runtime environment has been configured', () => {
+            const returnedValue = api.isRuntimeConfigured();
+            expect(returnedValue).to.be.true;
+        });
+
+        it ('should return false if the runtime environment has been configured', () => {
```
Comment (javiereguiluz) [inline test/index.js]: `has been configured` -> `has not been configured` ?

Comment (stof) [reply test/index.js]: indeed, bad copy paste

**Comment by stof** [comment]
> Also I've just seen some linting issues.

these don't seem to be part of the defined coding standards, as the linting is green on CI. But they are now fixed.

**Comment by stof** [comment]
should it be marked as an allowed failure then if you don't care about the fact it fail ? That would make the github status more useful.

**Comment by Lyrkan** [comment]
I don't know if allowing the job to fail is the right solution... it could be a temporary one though.

What I meant in my other comment is that it's not important if it fails for that specific reason but it could be for another one.

In my opinion the proper way to do it would be to ignore some checks (such as the ones that are based on a hash that we can't control) only when running that job, but I haven't had the time to look into that.

File: index.js [hunk 2]
```diff
@@ -1157,6 +1157,10 @@ class Encore {
         return this;
     }
 
+    isRuntimeConfigured() {
```
Comment (stof) [inline index.js]: Question here: should it be named `isRuntimeConfigured` or `isRuntimeEnvironmentConfigured` ? Other methods are taking about `RuntimeEnvironment`.

**Comment by Lyrkan** [comment]
Let's merge it that way, thanks @stof

**Pull request merged**